resources/assets/coffee/react/_components/user-avatar.coffee

```diff
@@ -28,10 +28,19 @@ bn = 'avatar'
 
   className = "#{bn} #{modifiers}"
 
-  if props.user.id?
+  avatar =
     div
       className: className
       style:
         backgroundImage: "url('#{props.user.avatar_url}')"
+
+  if props.user.id?
+    if props.user.id == currentUser.id && _.includes modifiers, 'profile'
```

Comment:
`props.modifiers`. `modifiers` is plain string (still works but not quite as intended).